projects/kit/components/multi-select/multi-select-native/multi-select-native.component.ts

```diff
@@ -57,7 +57,7 @@ export class TuiMultiSelectNative<T> {
 
     @Input()
     public set placeholder(placeholder: string) {
-        this.textfield.fillerSetter = placeholder;
+        this.textfield.filler.set(placeholder);
```

Comment:
Let use `tuiSetSignal` instead

```
import {tuiSetSignal} from '@taiga-ui/cdk/utils/miscellaneous';
```

Dont forget to update `projects/core/components/textfield/textfield.component.ts`

```diff
- public readonly filler = model('');
+ public readonly filler = input('');
```